Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.13
+FROM alpine:3.19
 MAINTAINER Rohith Jayawardene <gambol99@gmail.com>
```

Comment:
Small thing but shall we change this to ACP platform team email as maintainer?

Reply:
That's a good point, I'll get the maintainer updated